src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -786,25 +789,27 @@ export class BaseEditCourseForm extends React.Component {
             courseSubmitting={submitting}
             {...this.props}
           />
-          {this.getAddCourseRunButton(courseInReview, pristine, uuid)}
+          {this.getAddCourseRunButton(disabled, pristine, uuid)}
+          {editable &&
           <ButtonToolbar className="mt-3">
```

Comment:
Nit: Indent this after add in the `{editable &&` Unless that goes against eslint. I can't ever remember.

Reply:
I think eslint accepts both. I figured this was easier to visually follow, but I grant it makes it harder to see what is controlled by the editable.

Reply:
(Meaning, I'll make the indent change)